examples/nlp/multi_label_intent_slot_classification/conf/multi_label_intent_slot_classification_config.yaml

```diff
@@ -0,0 +1,111 @@
+# Intent and Slot classification with pretrained BERT models
+
+trainer:
+  gpus: 2 # the number of gpus, 0 for CPU
```

Comment:
Set the default number of GPUs to 1 since most people won't have more than one. I don't think Colab is offering more than one either.